### ISSUE TITLE
`ucd-theme-list-accordion`  - `aria-controls` and `aria-labelledby` have wrong index

### ISSUE THREAD

**Issue by spelkey-ucd** [ucd-library/ucdlib-theme#110]
should be ` aria-controls="accordion-${index + 1}-panel"`

and `aria-labelledby="accordion-${index - 1}"`
